fast_faults.py

```diff
@@ -121,6 +121,7 @@ def update_filters(self):
         options = [
             {'name': 'ok', 'channel': 'ca://${P}FFO:${FFO}:FF:${FF}:OK_RBV'},
             {'name': 'beampermitted', 'channel': 'ca://${P}FFO:${FFO}:FF:${FF}:BeamPermitted_RBV'},
+            # TODO: seems like a type below: ca://ca://... probably not doing any harm?
```

Comment:
I am going to take this off - for now I left it here to remind me to ask if this is `ca://ca://` on purpose? I am thinking it should not matter because it is probably parsing it fine even with double `ca` - but wanted to ask anyway

Reply:
The double ca looks really wrong to me. I bet pydm is just rolling its eyes and fixing it for us.

Reply:
>  I don't really understand how I would use the Photon Energy Ranges filter (or what as a user I'm expecting it to do) but I see the need for "something" in that space...

@slacAWallace looking back at this commend:
https://github.com/pcdshub/pmps-ui/issues/34#issuecomment-794346669 I think you meant that we only care about this when we don't have all the bits lit correct?

So instead of having the user check the individual bits and filter based on the decimal value of that (like i have it now), we should only care if `bitmask value < 65535`?